wp-content/themes/core/components/search_form/css/search-form.pcss

```diff
@@ -8,19 +8,41 @@
 	position: relative;
 }
 
+/* -----------------------------------------------------------------------------
+ * Search Form: Search Field
+ * ----------------------------------------------------------------------------- */
+
 .c-search__input {
 	width: 100%;
 	padding: var(--spacer-20) 76px var(--spacer-20) var(--spacer-30);
 	background: var(--color-white);
 	border: 1px solid var(--color-neutral-20);
 	color: var(--color-neutral-70);
+
+	/* CASE: Search Form in the site header */
+	.c-search--site-header & {
+		padding: 12px 42px;
```

Comment:
any chance we can get the sizing/spacing to match up with our spacer vars?

Reply:
This is a very unique case, one where I feel it's safe/ideal to use pixel values rather than a spacer var. If any changes are needed to the padding values here, it would need to be addressed based on the sizing of the search icon anyway.